src/main.cpp

```diff
@@ -237,7 +256,7 @@ Hitable *cornell_box()
     Material *green = new Lambertian(new ConstantTexture(c_green));
     Material *yellow = new Lambertian(new ConstantTexture(c_yellow));
 
-    Material *light = new DiffuseLight(new ConstantTexture(Color(150, 150, 150)));
+    Material *light = new DiffuseLight(new ConstantTexture(Color(1.0f)));
```

Comment:
Updated the light color definition with the new constructor.

By the way, check the image in the `README.MD` because something really weird is happening... I needed to update the light intensity, otherwise we ended up with something like this:

![image](https://user-images.githubusercontent.com/675471/41202614-b2ce505a-6cc3-11e8-8d03-5c190b269ee2.png)